### DISCUSSION TITLE
How do I get a badge

### DISCUSSION THREAD

**Discussion by Otherwa** [community/community#17816] [New to GitHub]
How??

**Comment by proarafat** [comment]
Please visit <a href="https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile#displaying-badges-on-your-profile">Here</a> for get badges and see how can you get GitHub Badges.

AND YOU ALSO SEE THIS REPOSITORY TO GET GitHub BADGES 
https://github.com/Schweinepriester/github-profile-achievements

**Reply by Otherwa** [reply]
Thank you 3000!
